README.md

```diff
@@ -1,161 +1,135 @@
-from vectara.factory import Factory
-client = Factory(profile="admin").build()
-```
-
-### Managers vs Direct API methods
-This API was generated by FERN, but in some cases you want to perform operations at a "higher level of abstraction"
-that may comprise multiple method calls. There's also many convenience operations we can perform which are useful that may eventually make it into
-the Vectara API. For example, we can URL encode the doc_id to support multiple languages which isn't (yet) in the
-FERN generated code.
-
-<img src="./resources/managers_seq.png" />
-
-To that end, we have built and tested the following managers on top of the direct API methods. They are located
-in the module vectara.managers. We have the following managers defined which
-can be found on the client.
-
-* [client.corpus_manager](./src/vectara/managers/corpus.py) - Provides abstractions over Corpus Methods
-* [client.upload_manager](./src/vectara/managers/upload.py) - Provides abstractions over file Uploads
-
-## Usage
+## Installation
 
-Using the [implicit configuration](#YAML-Configuration-Format) method we
-can create and run a query as follows.
+Install the library via pip:
 
-```python
-from vectara.factory import Factory
-
-client = Factory().build()
-client.query(
-    query="Am I allowed to bring pets to work?"
-)
+```bash
+pip install vectara
 ```
-More complete examples can be found in the [Getting Started notebooks](./examples/01_getting_started).
-
+## Getting Started
 
-## API Generated Documentation
+### API Generated Documentation
 API reference documentation is available [here](https://vectara.docs.buildwithfern.com/).
 
-## Exception Handling
+### Examples
+Complete examples can be found in the [Getting Started notebooks](./examples/01_getting_started).
+
+### Usage
+
+* Querying the corpora
+    ```python
+    from vectara import SearchCorporaParameters, Vectara
+    client = Vectara(
+        api_key="YOUR_API_KEY",
+        client_id="YOUR_CLIENT_ID",
+        client_secret="YOUR_CLIENT_SECRET",
+    )
+    
+    search = SearchCorporaParameters(
+        corpora=[...],
+        ...
+    )
+    generation = GenerationParameters(...)
+    
+    client.query(
+        query="Am I allowed to bring pets to work?",
+        search=search,
+        generation=generation
+        
+    )
+    ```
+ 
+* Starting the chat session
+   ```python
+   from vectara import SearchCorporaParameters, Vectara
+   client = Vectara(
+       api_key="YOUR_API_KEY",
+       client_id="YOUR_CLIENT_ID",
+       client_secret="YOUR_CLIENT_SECRET",
+   )
+    
+   search = SearchCorporaParameters(
+       corpora=[...],
+       ...
+   )
+   generation = GenerationParameters(...)
+   chat_params = ChatParameters(store=True)
+
+   session = client.create_chat_session(
+       search=search_params,
+       generation=generation_params,
+       chat_config=chat_params,
+   )
+   
+   response = session.chat(query="Tell me about machine learning.")
+   print(response.answer)
```

Comment:
Here please show how someone does a subsequent chat. i.e. using the returned Chat session for a new turn